pkg/batchmapper/examples/batchmap-flatmap/Makefile

```diff
@@ -0,0 +1,18 @@
+TAG ?= stable
+PUSH ?= false
+
+.PHONY: build
+build:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -o ./dist/flatmap-example main.go
+
+.PHONY: image-push
+image-push: build
+	docker buildx build -t "quay.io/numaio/numaflow-go/batch-map-flatmap:${TAG}" --platform linux/amd64,linux/arm64 --target flatmap . --push
```

Comment:
Please apply the changes in https://github.com/numaproj/numaflow-go/pull/134 here and add batchmap example to build-push.yaml so that it gets included in auto-update.

Reply:
@KeranYang I have added the changes, please verify once